contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -424,8 +424,11 @@ var ContentNodeCollection = BaseCollection.extend({
                     item.set('contentnode', node.id);
                     if(item.get('type') === 'input_question'){
                         item.get('answers').each( function(a){
-                            var value = numParser.parse(a.get('answer'))
-                            a.set('answer', value !== null && value.toString());
+                            var answer = a.get('answer');
+                            if (answer) {
+                                var value = numParser.parse(a.get('answer'))
+                                a.set('answer', value !== null && value.toString());
```

Comment:
Let's explicitly check if `value` is a string, instead of comparing to `null`.

Reply:
Actually, value won't be a string here, it'll be a number. We're just checking that numParser.parse didn't return a null value before setting the answer value. I'm not sure, but the explicit !== null check is probably in order to allow 0 as a value, which normally would get casted to false.

Reply:
Oh I see! the intent here is kinda hidden then. It might be easier to read that intent by expanding it into a whole if statement that looks like:

```
if (value.isNumber()) {
  a.set('answer', value.toString())
}
```

rather than comparing against null

Reply:
The bug this code is addressing happens when value IS null, though, which will fail in the above code because .isNumber() will be called on a null variable. I agree that more validation would be useful, but I don't know enough to know exactly what values are valid, in which case I prefer to leave the original approach in place until I know better how to change it.

Reply:
OK, I see what you mean -- let's make the minimal change then.